docs/en/ingest-management/elastic-agent/running-on-kubernetes-standalone.asciidoc

```diff
@@ -103,9 +101,9 @@ elastic-agent-hs4pb             1/1     Running   0          81m
 [float]
 ==== Autodiscover targeted Pods
```

Comment:
Same question as the previous article, but should autodiscover be hyphenated?

Reply:
No need to hyphenate.